test/k8sT/PoliciesNightly.go

```diff
@@ -30,8 +30,13 @@ var _ = Describe("NightlyPolicies", func() {
 
 	BeforeAll(func() {
 		kubectl = helpers.CreateKubectl(helpers.K8s1VMName(), logger)
+		_ = kubectl.Apply(helpers.DNSDeployment())
 
-		err := kubectl.CiliumInstall(helpers.CiliumDefaultDSPatch, helpers.CiliumConfigMapPatch)
+		// Deploy the etcd operator
+		err := kubectl.DeployETCDOperator()
```

Comment:
This can be done as a follow-up, but a lot of `BeforeAll` is becoming quite boilerplate heavy. We should factor this out into a generic test setup function that we can invoke in each `BeforeAll`.

Reply:
@ianvernon agreed, should be a "BeforeEachAll" or something like that